src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const population = document.getElementsByClassName('population');
+
+const totalPopulationElement = document
+  .getElementsByClassName('total-population')[0];
+const averagePopulationElement = document
+  .getElementsByClassName('average-population')[0];
```

Comment:
You should use only `querySelector` or `querySelectorAll`
They are the most flexible